components/asic/include/asic.h

```diff
@@ -0,0 +1,13 @@
+#ifndef ASIC_COMPONENT_H
+#define ASIC_COMPONENT_H
+#include <stdint.h>
+
+#define BLOCK_HEADER_SIZE 80
+
+struct block_header {
+    uint8_t bytes[BLOCK_HEADER_SIZE];
+};
```

Comment:
FYI: This is the "most idiomatic" way in C to have a function return a fixed size array - do it through a struct.

Reply:
are you calling me an idiot or an idiom?

Reply:
:rofl: - literally what chatGPT said: "most idiomatic"